eq-author-api/db/models/DynamoDB.js

```diff
@@ -0,0 +1,93 @@
+const dynamoose = require("dynamoose");
+
+let throughput = "ON_DEMAND";
+let questionnanaireTableName = "author-questionnaires";
+let questionnanaireVersionsTableName = "author-questionnaire-versions";
+
+if (process.env.DYNAMO_ENDPOINT !== "") {
```

Comment:
It assumes local if you provide a `DYNAMO_ENDPOINT` that seems a little weird. How does it derive the DYNAMO_ENDPOINT otherwise? Maybe this needs a better name - something like `DYNAMO_ENDPOINT_DEV`

Reply:
Its more of an override, when you run in AWS you don't need to provide one ill rename it to `DYNAMO_ENDPOINT_OVERRIDE`